src/components/dynamic/admin/dashboards/Admins.jsx

```diff
@@ -1,54 +1,7 @@
 "use client";
-import { useState } from "react";
-import Title from "../Title.jsx";
-import Filters from "../Filters.jsx";
-import Toolbar from "./Toolbar.jsx";
-import Table from "../Table.jsx";
-import {
-  FILTERS,
-  HEADERS,
-  STATUSES,
-  TAGS,
-} from "@/data/dynamic/admin/Admins.js";
+import Dashboard from "../Dashboard.jsx";
 
 const Admin = () => {
-  const [admin, setAdmin] = useState(null);
```

Comment:
these states need to stay here since they r unique to each dashboard and passed into the new comp